### PR TITLE
Fix function for show inline style in text only.

### PR DESCRIPTION
**Pull request by nacho87**
- **What kind of change does this PR introduce?** (Bug fix, feature,
  docs update, ...)
  You can add style inline to the api elements. 
- **What is the current behavior?** (You can also link to an open issue
  here)
  You can't  see in the frontend the element with style inline
- **What is the new behavior (if this is a feature change)?**

You can use style inline from the api
- **Does this PR introduce a breaking change?** (What changes might
  users need to make in their application due to this PR?)
- **Other information**:

### CODE REVIEW

File: lib/organisms/flexible-content/flexible-content.directive.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ function lnOFlexibleContent($compile, $injector, $sanitize) {
             parAttr = prefix + parAttr;
           }
 
-          var value = angular.isObject(row[key]) ? $sanitize(angular.toJson(row[key])) : row[key];
+          var value = angular.isObject(row[key]) ? $sanitize(angular.toJson(row[key])) : row[key].replace(/"/g, '\'');
```
Comment (mitogh) [inline lib/organisms/flexible-content/flexible-content.directive.js:42]: I think the problem also exists on strings with single quotes no?.

Comment (mitogh) [inline lib/organisms/flexible-content/flexible-content.directive.js:42]: Because you can scape a string by using `{{ expression }}``

Comment (andybar2) [inline lib/organisms/flexible-content/flexible-content.directive.js:42]: No @mitogh , just with double quotes because they break the attribute content :(

Comment (mitogh) [inline lib/organisms/flexible-content/flexible-content.directive.js:42]: Perfect, thanks for the heads up.

**Pull request merged**